typescript/postgis-express/README.md

```diff
@@ -36,28 +36,13 @@ npm install
 
 </details>
 
-### 2. Create and seed the database
-
-Run the following command to create your SQLite database file. This also creates the `User` and `Post` tables that are defined in [`prisma/schema.prisma`](./prisma/schema.prisma):
-
-```
-npx prisma migrate dev --name init
-```
-
-Now, seed the database with the sample data in [`prisma/seed.ts`](./prisma/seed.ts) by running the following command:
-
-```
-npx prisma db seed --preview-feature
-```
-
-
 ### 2. Setup PostgreSQL
```

Comment:
Can we use Prisma to set up the database using `migrate dev` or `db pull`?

Why does one have to go through the step of creating a database using SQL, `db pull` and then generate the client? 🤔

Reply:
In the seed file, let's add the `if not exists` when creating the tables

Reply:
`db pull` will not work for unsupported types. As for migrate, I couldn't get it working for automated tests so it would be great if you could check that out